### ISSUE TITLE
Trying to use Event() yields error in spawnv_passfds()

### ISSUE THREAD

**Issue by grahamrow** [uqfoundation/multiprocess#40]
Greetings. Programmed myself into a corner by getting something to work with `fork` on posix then realizing `spawn` was going to be a nightmare on windows. I have typically been subclassing `Process` directly and performing communication between processes with `Event`s, `Queue`s, and so on. Upon trying a drop in replacement with `multiprocess`, i.e.
```python
import multiprocess as mp
class Filter(mp.Process):
    def __init__(self):
        super(Filter, self).__init__()
        # Other stuff
        self.exit = mp.Event()
        # Other stuff
```
I see problems with creating the event. This ultimately traces to:
```
~/anaconda/lib/python3.6/site-packages/multiprocess/semaphore_tracker.py in ensure_running(self)
     59                 args = [exe] + util._args_from_interpreter_flags()
     60                 args += ['-c', cmd % r]
---> 61                 util.spawnv_passfds(exe, args, fds_to_pass)
     62             except:
     63                 os.close(w)

~/anaconda/lib/python3.6/site-packages/multiprocess/util.py in spawnv_passfds(path, args, passfds)
    393             args, [os.fsencode(path)], True, passfds, None, None,
    394             -1, -1, -1, -1, -1, -1, errpipe_read, errpipe_write,
--> 395             False, False, None)
    396     finally:
    397         os.close(errpipe_read)

TypeError: fork_exec() argument 4 must be tuple, not list
```
Which seems like a simple tuple/list confusion.

**Comment by shayegano** [comment]
Came across this thread after running into this myself.

**Background:** A few months back, a [Python patch](https://bugs.python.org/issue30065) was made to update _posixsubprocess validation. That patch was applied [here](https://github.com/python/cpython/commit/c97c1914f401359f2a7e6c8e0364e71ad9fb5bc8#diff-f514e361152f1443fb488f6cf4dbdb48), and also included the tuple conversion fix in multiprocessing/util.py. A similar fix was then applied to multiprocess/util.py [here](https://github.com/uqfoundation/multiprocess/commit/e290d4c59d7540c924b11d0c8bfe7c1647487087#diff-cf1f138aeb70a90b2c6129dc1eabcecb).

Unfortunately, the latest release, multiprocess-0.70.5, came before the latter patch, so installing multiprocess via a package manager (as I did, using pip) causes the above error. 

**The fix:** just install latest multiprocess from source using `python setup.py install` :)

**Comment by mmckerns** [comment]
@shayegano: that's correct.  The fix for `3.6` is as you point out above, and additionally for `3.5` in 65d79d2b27b8fd56686c2b9d17edbd54be9a0764.

I'm going to close this, but @grahamrow, if this is not a sufficient resolution for you, let me know.
